crates/rooch-key/src/key_derive.rs

```diff
@@ -21,6 +21,9 @@ pub const DERIVATION_PATH_PURPOSE_SCHNORR: u32 = 44;
 pub const DERIVATION_PATH_PURPOSE_ECDSA: u32 = 54;
 pub const DERIVATION_PATH_PURPOSE_SECP256R1: u32 = 74;
 pub const DERIVATION_PATH_PURPOSE_BIP86: u32 = 86;
+pub const SIGNATURE_SCHEME_FLAG_ED25519: u8 = 0x00;
+pub const SIGNATURE_SCHEME_FLAG_SECP256K1: u8 = 0x01;
```

Comment:
Directly use the SignatureScheme in crypto.rs?

https://github.com/rooch-network/rooch/blob/477f8b7c4677d8c71ce2ed6f54a2eb341b488281/crates/rooch-types/src/crypto.rs#L53-L59

Reply:
Resolved using SignatureScheme.